test/models/database/ShortServiceRefundsStorageAnswersSpec.scala

```diff
@@ -21,10 +21,10 @@ import org.scalatest.wordspec.AnyWordSpecLike
 
 class ShortServiceRefundsStorageAnswersSpec extends AnyWordSpecLike {
 
-  "fromJourneyAnswers" should {
-    "create ShortServiceRefundsStorageAnswers" in {
-      val result = ShortServiceRefundsStorageAnswers.fromJourneyAnswers(shortServiceRefundsAnswers)
-      assert(result === shortServiceRefundsCtxStorageAnswers)
-    }
-  }
+//  "fromJourneyAnswers" should {
```

Comment:
let's add TODO if we plan to fix it in some point. In future we'll clean up all TODOs in the project.